script.js

```diff
@@ -21,4 +21,170 @@ navListEl.forEach((element) => {
+        <li>${projects[i].technologies[0]}</li>
+        <li>${projects[i].technologies[1]}</li>
+        <li>${projects[i].technologies[2]}</li>
+      </ul>
+    </div>
+    <button type="button" id="btn-${i}">See Project</button>
+  </div>
+  `;
+}
+
+const modal = document.getElementById('modal1');
+
+for (let j = 0; j < projects.length; j += 1) {
+  const seeBtn = document.querySelectorAll(`[id=btn-${j}]`);
+  seeBtn.forEach((bt) => {
+    bt.addEventListener('click', () => {
+      modal.style.display = 'block';
+      modal.innerHTML = `
+      <div class="popup-container block-popup">
+      <div id="popup-modal" class="pop-content">
+        <div class="headline-popup">
+          <h1>${projects[j].name}</h1>
+          <a href="#" role="button" onclick="closePopup()">
+            <div class="popup-clsbtn">x</div>
+          </a>
+        </div>
+      </div>
+      <ul class="small-tags">
+        <li class="canopy-sm-tag">${projects[j].des[0]}</li>
+        <li>
+          <div class="round-popup"></div>
+        </li>
+        <li class="second-sm-tag">${projects[j].des[1]}</li>
+        <li>
+          <div class="round-popup"></div>
+        </li>
+        <li class="second-sm-tag">${projects[j].des[2]}</li>
+      </ul>
+      <div class="popup-image-container">
+        <img src="${projects[j].img}" alt="pop-image" />
+      </div>
+      <div class="buttom-content-container">
+        <div class="paragraph-description">
+          <p class="paragraph-description-desktop">
+            ${projects[j].desc1}
+          </p>
+        </div>
+        <div class="action-content">
+          <div class="popup-tags">
+            <ul class="first">
+            <li>${projects[j].technologies[0]}</li>
+            <li>${projects[j].technologies[1]}</li>
+            <li>${projects[j].technologies[2]}</li>
+            </ul>
+            <ul class="second">
+            <li>${projects[j].technologies1[0]}</li>
+            <li>${projects[j].technologies1[1]}</li>
+            <li>${projects[j].technologies1[2]}</li>
+            </ul>
+          </div>
+          <div class="divider-popup">
+            <img src="./images/Vector 4.png" alt="Divider popup" />
+          </div>
+          <div class="popup-button-action">
+            <button type="submit">
+              <div class="source-code">
+                <a href="">See live </a>
+              </div>
+              <div class="deploy-icon">
+                <img src="./images/deploy.svg" alt="deployment icon" class="deploy-icon-popup" />
+              </div>
+            </button>
+            <button class="submit-github">
+              <div class="live-version">
+                <a href="">See Source </a>
+              </div>
+              <div class="github-img">
+                <img src="./images/github-popup.svg" alt="github icon" class="github-icon-popup" />
+              </div>
+
+            </button>
+          </div>
+        </div>
+      </div>
+    </div>
```

Comment:
- [ ] The modal is very similar to the Figma design, but it's currently too big not fitting on any screen on the desktop and mobile view. Kindly make them responsive so they can fit on every screen and the user is able to see all the content of the modal without the need of scrolling up and down for a better user experience. (You could reduce the description, the image size, etc.)
 - Your design:
 - 

<img width="1150" alt="image" src="https://user-images.githubusercontent.com/61919617/216031176-b73b671b-4006-44a2-8cbf-b16ad25ec880.png">
<img width="473" alt="image" src="https://user-images.githubusercontent.com/61919617/216031241-899ad224-e3c2-4aeb-859e-bbf0dbaef5fe.png">